system/doc/system_principles/create_target.xmlsrc

```diff
@@ -263,6 +263,14 @@ os> <input>/usr/local/erl-target/bin/erl -boot /usr/local/erl-target/releases/FI
       current directory create not only the file <c>mysystem.rel</c>,
       but also file <c>sys.config</c>, the latter file is tacitly
       put in the appropriate directory.</p>
+    <p>However, it can also be convenient to replace variables in
+       within a <c>sys.config</c> on the target after unpacking but
+       before running the release. If you have a <c>sys.config.src</c>
+       it will be included and is not required to be a valid Erlang term
+       file like <c>sys.config</c>. Before running the release you must
+       have a valid <c>sys.config</c> in the same directory, so using
+       <c>sys.config.src</c> requires having some tool to populate what is
+       needed and write <c>sys.config</c> to disk before booting the release.
```

Comment:
Missing end tag 'p'

Reply:
Please make sure the documentation can be built.

Reply:
Oh, my bad, I thought the CI checks included that :(. Will do.

Reply:
We've tried having it, but iirc Travis keeps timing out with it.

Reply:
@garazdawi It is probably their requirement to have console output when there is a pause, so if a progress bar was added, it should work without a Travis timeout.

Reply:
It was the 60 min timer that was hit. Sometimes our Travis jobs seem to take about 3 times as long, not sure why, maybe it ends up on a super busy host. Our current builds time out when that happens, which is maybe 1 in 100 builds. When I added docs, that number went up to about 1 in 5 builds or so. So alas no doc builds. We would be very happy if someone fixed it so that it works.

/end of pr hijack